numpy/ma/core.py

```diff
@@ -6824,7 +6824,13 @@ def argsort(a, axis=np._NoValue, kind=None, order=None, endwith=True, fill_value
 argsort.__doc__ = MaskedArray.argsort.__doc__
 
 def sort(a, axis=-1, kind=None, order=None, endwith=True, fill_value=None):
-    "Function version of the eponymous method."
+    """
+    Equivalent to self.copy.sort()
```

Comment:
Maybe start with `Return a sorted copy of the masked array.` And then continue with the equivalent to `...` and ```refer to `numpy.ma.MaskedArray.sort` ``` for the full documentation.
Also `self.copy().sort()`, although I am not sure it actually helps much to spell it out.

Reply:
Agreed, made the change.